Cannot download files on Android 11

**Describe the bug**
Files cannot be downloaded on Android 11.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to Files -> Documents
2. Navigate to a file
3. Click on the file to start downloading it
4. The app returns to main menu and the file is not downloaded

**Expected behavior**
The file is downloaded.

**Screenshots**
N/A

**Smartphone (please complete the following information):**
 - Device: Google Pixel 2 XL
 - OS: Android
 - Version 11

**Additional context**

`2020-09-29 17:05:04.599 23986-24095/es.ugr.swad.swadroid E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #1
    Process: es.ugr.swad.swadroid, PID: 23986
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$4.done(AsyncTask.java:415)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
        at java.util.concurrent.FutureTask.run(FutureTask.java:271)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:923)
     Caused by: java.lang.IllegalStateException: Not one of standard directories: SWADroid
        at android.os.Parcel.createExceptionOrNull(Parcel.java:2381)
        at android.os.Parcel.createException(Parcel.java:2357)
        at android.os.Parcel.readException(Parcel.java:2340)
        at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:190)
        at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:142)
        at android.content.ContentProviderProxy.call(ContentProviderNative.java:732)
        at android.content.ContentProviderClient.call(ContentProviderClient.java:603)
        at android.content.ContentProviderClient.call(ContentProviderClient.java:591)
        at android.app.DownloadManager$Request.setDestinationInExternalPublicDir(DownloadManager.java:576)
        at es.ugr.swad.swadroid.modules.downloads.DownloadFactory.downloadFile(DownloadFactory.java:85)
        at es.ugr.swad.swadroid.modules.downloads.FileDownloaderAsyncTask.doInBackground(FileDownloaderAsyncTask.java:70)
        at es.ugr.swad.swadroid.modules.downloads.FileDownloaderAsyncTask.doInBackground(FileDownloaderAsyncTask.java:42)
        at android.os.AsyncTask$3.call(AsyncTask.java:394)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:923) `

--- a/SWADroid/src/main/java/es/ugr/swad/swadroid/Constants.java
+++ b/SWADroid/src/main/java/es/ugr/swad/swadroid/Constants.java
@@ -324,7 +324,7 @@ public class Constants {
      */
     public static final String DIRECTORY_SWADROID = "SWADroid";
     public static final String DOWNLOADS_PATH =
-            Environment.getExternalStorageDirectory().getAbsolutePath() + File.separator + DIRECTORY_SWADROID;
+            Environment.DIRECTORY_DOWNLOADS + File.separator + DIRECTORY_SWADROID;
 
     /**
      * Username template for messages

--- a/SWADroid/src/main/java/es/ugr/swad/swadroid/modules/downloads/DownloadFactory.java
+++ b/SWADroid/src/main/java/es/ugr/swad/swadroid/modules/downloads/DownloadFactory.java
@@ -21,20 +21,14 @@ package es.ugr.swad.swadroid.modules.downloads;
 import android.annotation.TargetApi;
 import android.app.DownloadManager;
 import android.content.Context;
-import android.content.Intent;
-import android.content.pm.PackageManager;
-import android.content.pm.ResolveInfo;
 import android.net.Uri;
 import android.os.Build;
+import android.os.Environment;
 import android.util.Log;
-import android.widget.Toast;
 
-import java.io.File;
-import java.util.List;
 import java.util.Locale;
 
 import es.ugr.swad.swadroid.Constants;
-import es.ugr.swad.swadroid.utils.Utils;
 
 /**
  * Class for manage file downloads
@@ -60,56 +54,24 @@ public class DownloadFactory {
 			String description) {
 		
 		Uri uri = Uri.parse(url);	 
-	    DownloadManager managerHoneycomb;
-		DownloadManager.Request requestHoneycomb;
-	    es.ugr.swad.swadroid.modules.downloads.DownloadManager managerGingerbread;
-	    es.ugr.swad.swadroid.modules.downloads.DownloadManager.Request requestGingerbread;
-	    
-	    //Create destination directory if not exists
-	    File downloadDirectory = new File(Constants.DOWNLOADS_PATH);
-	    if (!downloadDirectory.exists()){
-	    	if(downloadDirectory.mkdir()) {
-	            Log.i(TAG, "Created directory " + Constants.DOWNLOADS_PATH);
-            } else {
-                Log.e(TAG, "Error creating directory " + Constants.DOWNLOADS_PATH);
-                Toast.makeText(context, "Error creating directory " + Constants.DOWNLOADS_PATH, Toast.LENGTH_LONG).show();
-                return false;
-            }
-        }
+	    DownloadManager manager;
+		DownloadManager.Request request;
 
-		managerHoneycomb = (DownloadManager) context.getSystemService(Context.DOWNLOAD_SERVICE);
-		requestHoneycomb = new DownloadManager.Request(uri);
+		manager = (DownloadManager) context.getSystemService(Context.DOWNLOAD_SERVICE);
+		request = new DownloadManager.Request(uri);
 
-		requestHoneycomb.setDescription(title);
-		requestHoneycomb.setTitle(description);
-		requestHoneycomb.setDestinationInExternalPublicDir(Constants.DIRECTORY_SWADROID, fileName);
+		request.setDescription(title);
+		request.setTitle(description);
+		request.setDestinationInExternalPublicDir(Environment.DIRECTORY_DOWNLOADS, Constants.DIRECTORY_SWADROID + "/" + fileName);
 
-		//DownloadManager HONEYCOMB
-		Log.i(TAG, "Downloading file " + fileName + " with DownloadManager >= HONEYCOMB");
+		Log.i(TAG, "Downloading file " + fileName);
 
-		requestHoneycomb.allowScanningByMediaScanner();
-		requestHoneycomb.setNotificationVisibility(DownloadManager.Request.VISIBILITY_VISIBLE_NOTIFY_COMPLETED);
+		request.allowScanningByMediaScanner();
+		request.setNotificationVisibility(DownloadManager.Request.VISIBILITY_VISIBLE_NOTIFY_COMPLETED);
 
-		managerHoneycomb.enqueue(requestHoneycomb);
+		manager.enqueue(request);
 	
 	    return true;
-	}
-
-	/**
-	 * @param context used to check the device version and DownloadManager information
-	 * @return true if the download manager is available
-	 */
-	public static boolean isDownloadManagerAvailable(Context context) {
-	    try {
-	        Intent intent = new Intent(Intent.ACTION_MAIN);
-	        intent.addCategory(Intent.CATEGORY_LAUNCHER);
-	        intent.setClassName("com.android.providers.downloads.ui", "com.android.providers.downloads.ui.DownloadList");
-	        List<ResolveInfo> list = context.getPackageManager().queryIntentActivities(intent,
-	                PackageManager.MATCH_DEFAULT_ONLY);
-	        return list.size() > 0;
-	    } catch (Exception e) {
-	        return false;
-	    }
 	}
 
 	/**

--- a/SWADroid/src/main/java/es/ugr/swad/swadroid/modules/downloads/DownloadFactory.java
+++ b/SWADroid/src/main/java/es/ugr/swad/swadroid/modules/downloads/DownloadFactory.java
@@ -26,6 +26,7 @@ import android.os.Build;
 import android.os.Environment;
 import android.util.Log;
 
+import java.io.File;
 import java.util.Locale;
 
 import es.ugr.swad.swadroid.Constants;
@@ -62,7 +63,7 @@ public class DownloadFactory {
 
 		request.setDescription(title);
 		request.setTitle(description);
-		request.setDestinationInExternalPublicDir(Environment.DIRECTORY_DOWNLOADS, Constants.DIRECTORY_SWADROID + "/" + fileName);
+		request.setDestinationInExternalPublicDir(Environment.DIRECTORY_DOWNLOADS, Constants.DIRECTORY_SWADROID + File.separator + fileName);
 
 		Log.i(TAG, "Downloading file " + fileName);
 

